test/linter.js

```diff
@@ -116,15 +122,21 @@ describe('AutoRest Linter validation:', function () {
             return prFile.startsWith(configDir) && prFile.indexOf('examples') === -1 && prFile.endsWith('.json');
           }).forEach(prFileInConfigFile => {
             console.warn(`WARNING: Configuration file not found for file: ${prFileInConfigFile}, running validation rules against it in individual context.`);
-            execLinterCommand(`--input-file=${prFileInConfigFile}`);
+            errorsFound = errorsFound && execLinterCommand(`--input-file=${prFileInConfigFile}`);
```

Comment:
should this be an OR statement? same below

Reply:
well with OR we would just have the old behavior ( once it founds errors it would stop processing further ). I think the intention of the processing to continue. In that case just execute the command seperatley and && on the result

Reply:
why will it continue ? if we put `||` once errorsFound becomes `true` ( i.e. one of them fails) then it will not execute `execLinterCommand` because of the shortcircuit logic right ?  ( for files here for tags afterwards )